### PR TITLE
SDI-88: 🎨 Convert int tests to typescript

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,8 +1,6 @@
 node_modules
+public
 assets
-package-lock.json
```
Comment (petergphillips) [inline .eslintignore:3]: copied from template typescript project

File: .eslintrc.json [hunk 2]
```diff
@@ -4,11 +4,94 @@
...
+            "argsIgnorePattern": "res|next|^err|_",
+            "ignoreRestSiblings": true
+          }
+        ],
+        "@typescript-eslint/semi": 0,
+        "import/no-unresolved": "error",
+        "prettier/prettier": [
+          "error",
+          {
+            "trailingComma": "all",
```
Comment (petergphillips) [inline .eslintrc.json:51]: copied from template typescript project and changed from `es5` to `all`

File: .circleci/config.yml [hunk 3]
```diff
@@ -131,9 +131,9 @@ jobs:
       - store_test_results:
           path: test-results
       - store_artifacts:
-          path: integration-tests/videos
```
Comment (petergphillips) [inline .circleci/config.yml:134]: changed to match typescript template

File: .eslintrc.json [hunk 4]
```diff
@@ -4,11 +4,94 @@
...
+            "singleQuote": true,
+            "printWidth": 120,
+            "semi": false
```
Comment (petergphillips) [inline .eslintrc.json:54]: and remove `arrowParens` avoid to match existing

File: .eslintrc.json [hunk 5]
```diff
@@ -4,11 +4,94 @@
...
+      "ignorePackages",
+      {
+        "js": "never",
+        "mjs": "never",
+        "jsx": "never",
+        "ts": "never",
+        "tsx": "never"
+      }
+    ],
+    "comma-dangle": ["error", "always-multiline"],
```
Comment (petergphillips) [inline .eslintrc.json:85]: preserved the much better `comma-dangle` setting compared to the typescript template project too

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**